gui/CMakeLists.txt

```diff
@@ -1,38 +1,47 @@
-find_package(Qt5Widgets)
-if(Qt5Widgets_FOUND)
-else()
-find_package(Qt4)
-include(${QT_USE_FILE})
-endif()
-
-include_directories(../include/gmapping ../ ../include/gmapping/log/)
+include_directories(.. ../include/gmapping ../include/gmapping/log)
```

Comment:
We should first check for find_package(Qt5Widgets) and return if not found: if we add this GUI for gmapping, it will be made into it's own package. For now, we allow to build it in here.